apps/nycbus/nyc_bus.star

```diff
@@ -98,40 +99,80 @@ def main(config):
         )
 
     return render.Root(
+        delay = 75,
         child = render.Column(
             expanded = True,
-            main_align = "space_evenly",
+            main_align = "start",
             children = [
                 build_row(journeys[0]),
                 render.Box(
                     width = 64,
                     height = 1,
-                    color = "#aaa",
+                    color = "#666",
                 ),
                 build_row(journeys[1]),
             ],
         ),
     )
 
 def build_row(journey):
+    # Only match names of bus lines that we know won't fit
+    multi_line = re.compile("([A-Za-z]+)([0-9]+)([\\/ -])([A-Za-z0-9]+)")
+    match = multi_line.match(journey["line_name"])
+    if len(journey["line_name"]) > 4 and len(match):
+        _, borough, first, sep, second = match[0]
+
+        # Lines like "M55/56" should translate to -> M55, M56
+        if sep in ("\\", "/"):
+            parts = [borough + first, borough + second]
+        elif sep == "-":
+            # Lines like "Bx41-SBS"
+            parts = journey["line_name"].split("-")
+        else:
+            # Lines like "M44 Ltd"
```

Comment:
I guess this example is relevant to my top level question about whether we need the sbs or ltd.

Also I just searched and confirmed there are sbs and non variants for the same lines, like the bx12

Reply:
Admittedly I'm not sure if you would see "Ltd" come through your feed - I included it for completeness based on the table I was referencing here: https://metro.fandom.com/wiki/List_of_bus_routes_in_Manhattan

Same goes for M55/56 - this might just come through as whichever of the two arrives. If "-SBS" is the only possible suffix I'm open to simplifying this bit if you'd prefer!

Reply:
I suspect that the ltd won't be in the bus code we're using here but I'm fine to leave this as is.